api-reference/10 UI Components/GridBase/1 Configuration/columnChooser/selection.md

```diff
@@ -5,7 +5,7 @@ inheritsType: ColumnChooserSelectionConfig
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures the column chooser's selection.
```

Comment:
Again, this is a vague phrase: "column chooser's selection". What do we select? Do we select the column chooser? Do we select columns? Is it a UI feature or runtime functionality? It's hard to say. 

```suggestion
Configures column selection functionality within the column chooser.
```